business_register/filters.py

```diff
@@ -14,6 +16,14 @@ class Meta:
         }
 
 
+class FopFilterSet(filters.FilterSet):
+    fullname = filters.CharFilter(lookup_expr='iexact')
+    address = filters.CharFilter(lookup_expr='icontains')
+
+    class Meta:
+        model = Fop
+
```

Comment:
you need to add blank list of fields
```suggestion
    class Meta:
        model = Fop
        fields = ()
```

Reply:
done